package.json

```
@@ -52,7 +53,31 @@
         }
       ]
     },
+    "customEditors": [
+      {
+        "viewType": "cfg.editor",
+        "displayName": "Cfg Editor",
+        "selector": [
+          {
+            "filenamePattern": "*.cfg"
+          }
+        ],
+        "priority": "default"
+      }
+    ],
     "commands": [
+      {
+        "command": "onevscode.flip-cfg-to-ui",
+        "title": "Open cfg (UI)",
```

```suggestion
        "title": "Open cfg in graphical editor",
```
Maybe there would be better title to user.

---

I will change it :)
Just FYI, this came from vscode builtin UserSetting editor :D